libs/designsystem/src/lib/components/charts/chart-config/chart-stock-config/chart-stock-config.ts

```diff
@@ -0,0 +1,198 @@
+import {
+  BarController,
+  BarElement,
+  CategoryScale,
+  Chart,
+  ChartConfiguration,
+  ChartType,
+  ChartTypeRegistry,
+  Filler,
+  LinearScale,
+  LineController,
+  LineElement,
+  PointElement,
+  TimeScale,
+  Tooltip,
+  TooltipCallbacks,
+  TooltipItem,
+  TooltipLabelStyle,
+  TooltipOptions,
+} from 'chart.js';
+import annotationPlugin from 'chartjs-plugin-annotation';
+import { ColorHelper, DesignTokenHelper } from '../../../../helpers';
+import { ChartLocale } from '../../shared';
+import { ChartBaseConfig } from '../chart-base-config';
+
+import {
+  dataLabelsPluginConfig,
+  getDataLabelPosition,
+} from './data-label-position/data-label-position';
+import { getVerticalLinePluginConfig } from './vertical-line-plugin/vertical-line-plugin';
+
+const { getThemeColorHexString, getThemeColorRgbString } = ColorHelper;
+const { fontSize } = DesignTokenHelper;
+
+/**
+ * This class is still experimental. Feel free to use it, but please note that it is still subject to breaking changes.
+ */
+export class StockChartConfig extends ChartBaseConfig {
+  private static STOCK_CHART_LOCALE_DEFAULT: ChartLocale = 'en-US';
+
+  private static scales = [CategoryScale, LinearScale, TimeScale];
+  private static elements = [BarElement, LineElement, PointElement];
+  private static controllers = [BarController, LineController];
+  private static plugins = [annotationPlugin, Filler, Tooltip];
+  private static registeredElements = [
+    ...this.scales,
+    ...this.controllers,
+    ...this.elements,
+    ...this.plugins,
+  ];
+
+  private static pluginsRegistered = false;
+
+  public static registerPlugins() {
+    if (StockChartConfig.pluginsRegistered) {
+      return;
+    }
+
+    this.registeredElements.forEach((scale) => {
+      Chart.register(scale);
+    });
+    StockChartConfig.pluginsRegistered = true;
+  }
+
+  public static basicConfig: ChartConfiguration<ChartType> = {
+    type: 'line',
+    data: {
+      datasets: [],
+    },
+    options: {
+      responsive: true,
+      animation: {
+        duration: 0,
+      },
+      layout: {
+        padding: {
+          left: 20,
+          right: 20,
+          top: 30,
+          bottom: 0,
```

Comment:
Something worth thinking about, but which should not be handled now:
Would be great to store some of these magic numbers as design tokens instead, to preserve similar values across different chart configs.